src/provider/application.js

```diff
@@ -260,8 +260,15 @@ class Application extends EventEmitter {
   }
 
   unload() {
+    // Need to check for href attributes to avoid flickering.
+    const attributeCheck = document.activeElement.getAttribute('href');
+    let attributeCheck_href = false;
+    if(attributeCheck.includes('tel') || attributeCheck.includes('mailto') || attributeCheck.includes('fax') || 
```

Comment:
Could we have a unit test for this situation?

Reply:
will update unit tests!

Reply:
with the new code change, this won't be necessary